subsys/pm/policy/Kconfig

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-choice
+choice PM_POLICY
```

Comment:
Please revert this change if it is not necessary.

Reply:
We were not able to define PM_POLICY_APP (if it is not named choice) because of this issue https://github.com/zephyrproject-rtos/zephyr/issues/6948